test/redirects.js

```diff
@@ -38,6 +38,13 @@ test.before('setup', async () => {
 		response.end();
 	});
 
+	http.on('/to-port-80', (request, response) => {
```

Comment:
```suggestion
	http.on('/redirect-to-port-80', (request, response) => {
```

Reply:
And why not keep this endpoint inside the test itself?

Reply:
Well, if we want to move the route into the specific test, then we should do it everywhere, not only here.

Also, that would result in having two servers in one test function.

Reply:
> Well, if we want to move the route into the specific test, then we should do it everywhere, not only here.

Yes, but that's an insane amount of work. I think we should at least do so for new tests. Better to do it iteratively.

> Also, that would result in having two servers in one test function.

Why is that a problem?

Reply:
> Why is that a problem?

The code becomes less readable.

> Yes, but that's an insane amount of work. I think we should at least do so for new tests. Better to do it iteratively.

Ok so.

Reply:
> The code becomes less readable.

I would actually argue for the opposite. By having all the related code in the test you can fully understand what it does. If the test calls the server code, you need to scroll up or look in another file to see what the server part of the test does. Also, having shared tests makes it very easy to reuse endpoints for different tests, which means if you want to change the logic of one test and endpoint, you need to change many tests.